mdk/src/main/resources/META-INF/mods.toml

```diff
@@ -7,6 +7,8 @@
 modLoader="javafml" #mandatory
 # A version range to match for said mod loader - for regular FML @Mod it will be the forge version
 loaderVersion="[31,)" #mandatory This is typically bumped every Minecraft version by Forge. See our download page for lists of versions.
+# The license for you mod. This allows for easier comprehension of your redistributive properties
+license="CHOOSE YOUR LICENSE" #optional
```

Comment:
The literal string "CHOOSE YOUR LICENSE" isn't a valid licence. Please default to the same string that MDKs on newer MC versions use.